Kudu.Core/Deployment/WellKnownEnvironmentVariables.cs

```diff
@@ -36,5 +36,6 @@ internal static class WellKnownEnvironmentVariables
         public const string SiteBitness = "SITE_BITNESS";
 
         public const string GoWebConfigTemplate = "GO_WEB_CONFIG_TEMPLATE";
+        public const string SelectLatestVersionCommandKey = "SELECT_LATEST_VERSION_CMD";
```

Comment:
right, it is not Go specific. i will add some more command to describe it. basically it only look at the folders from a given path (NOT recursive), and find the largest version.